### PR TITLE
[Stream] Misc updates from product team

### PR DESCRIPTION
**Pull request by tsmith512**
Four minor updates:

- Typos
- Pricing clarifications
- Note on go-tus (ref #11969)
- Concurrency limit explanation update

### CODE REVIEW

File: src/content/docs/stream/faq.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 * By default, a video upload can be at most 30 GB.
 
-* By default, you can have up to 120 videos in the `inprogress`, `queued` or `downloading` state at the same time. Videos in the `error`, `ready` or `pendingupload` state do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
+* By default, you can have up to 120 videos queued or being encoded simultaneously. Videos in the `ready` status are playable but may still be encoding certain quality levels until the `pctComplete` reaches 100. Videos in the `error`, `ready`, or `pendinguipload` state and do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
```
Comment (third774) [inline src/content/docs/stream/faq.mdx]:
```suggestion
* By default, you can have up to 120 videos queued or being encoded simultaneously. Videos in the `ready` status are playable but may still be encoding certain quality levels until the `pctComplete` reaches 100. Videos in the `error`, `ready`, or `pendingupload` state and do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
```

File: src/content/docs/stream/pricing.mdx [hunk 2]
```diff
@@ -31,9 +31,9 @@ Storage is not consumed by:
 - Deleted videos
 - Downloadable files generated for [MP4 Downloads](/stream/viewing-videos/download-videos/)
 
-Storage consumption is rounded up to the second of video duration. File size does not matter.
+Storage consumption is rounded up to the second of video duration. File size does not matter. Video stored in Stream does not incur additional storage fees from other storage products like R2.
```
Comment (third774) [inline src/content/docs/stream/pricing.mdx]:
```suggestion
Storage consumption is rounded up to the second of video duration. File size does not matter. Video stored in Stream does not incur additional storage fees from other storage products such as R2.
```

Comment (third774) [reply src/content/docs/stream/pricing.mdx]: Thought that "such as" could be better here so as not to potentially imply that R2 does incur additional fees from other storage products? Also, not a strong feeling on this though.

**Review by third774 — APPROVED** [review]

File: src/content/docs/stream/faq.mdx [hunk 3]
```diff
@@ -25,7 +25,7 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 * By default, a video upload can be at most 30 GB.
 
-* By default, you can have up to 120 videos in the `inprogress`, `queued` or `downloading` state at the same time. Videos in the `error`, `ready` or `pendingupload` state do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
+* By default, you can have up to 120 videos queued or being encoded simultaneously. Videos in the `ready` status are playable but may still be encoding certain quality levels until the `pctComplete` reaches 100. Videos in the `error`, `ready`, or `pendingupload` state and do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
```
Comment (bkrebsbach) [inline src/content/docs/stream/faq.mdx]: I think the `and` after `pendingupload state` should be removed

Comment (tsmith512) [reply src/content/docs/stream/faq.mdx]: Ah, good catch. Thanks!

**Review by bkrebsbach — APPROVED** [review]

File: src/content/docs/stream/faq.mdx [hunk 4]
```diff
@@ -25,7 +25,7 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 * By default, a video upload can be at most 30 GB.
 
-* By default, you can have up to 120 videos in the `inprogress`, `queued` or `downloading` state at the same time. Videos in the `error`, `ready` or `pendingupload` state do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
+* By default, you can have up to 120 videos queued or being encoded simultaneously. Videos in the `ready` status are playable but may still be encoding certain quality levels until the `pctComplete` reaches 100. Videos in the `error`, `ready`, or `pendingupload` state do not count toward this limit. If you need the concurrency limit raised, please [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
```
Comment (dcpena) [inline src/content/docs/stream/faq.mdx]:
```suggestion
* By default, you can have up to 120 videos queued or being encoded simultaneously. Videos in the `ready` status are playable but may still be encoding certain quality levels until the `pctComplete` reaches 100. Videos in the `error`, `ready`, or `pendingupload` state do not count toward this limit. If you need the concurrency limit raised, [contact Cloudflare support](/support/contacting-cloudflare-support/) explaining your use case and why you would like the limit raised.
```

File: src/content/docs/stream/pricing.mdx [hunk 5]
```diff
@@ -31,9 +31,9 @@ Storage is not consumed by:
 - Deleted videos
 - Downloadable files generated for [MP4 Downloads](/stream/viewing-videos/download-videos/)
 
-Storage consumption is rounded up to the second of video duration. File size does not matter.
+Storage consumption is rounded up to the second of video duration. File size does not matter. Video stored in Stream does not incur additional storage fees from other storage products such as R2.
```
Comment (dcpena) [inline src/content/docs/stream/pricing.mdx]:
```suggestion
Storage consumption is rounded up to the second of video duration, and file size does not matter. Video stored in Stream does not incur additional storage fees from other storage products such as R2.
```

File: src/content/docs/stream/pricing.mdx [hunk 6]
```diff
@@ -31,9 +31,9 @@ Storage is not consumed by:
...
 
-**If you run out of storage**, you will not be able to upload new videos or start new live streams until you purchase more storage or delete videos.
+**If you run out of storage**, you will not be able to upload new videos or start new live streams until you purchase more storage or delete videos. Enterprise customers may continue to upload new content beyond their contracted quota without interruption.
```
Comment (dcpena) [inline src/content/docs/stream/pricing.mdx]: I think this better suited as a note instead of leading with weighted text.

**Review by dcpena — APPROVED** [review]

**Pull request merged**